reglob/reglob.js

```diff
@@ -54,38 +54,45 @@ function Reglob(glob, opts) {
     case "?":
       if (extended) {
         reStr += ".";
-      break;
+        break;
       }
 
     case "[":
     case "]":
       if (extended) {
         reStr += c;
-      break;
+        break;
       }
 
     case "{":
       if (extended) {
         inGroup = true;
-      reStr += "(";
-      break;
+        reStr += "(";
+        break;
       }
 
     case "}":
       if (extended) {
         inGroup = false;
-      reStr += ")";
-      break;
+        reStr += ")";
+        break;
       }
 
     case ",":
       if (inGroup) {
         reStr += "|";
-      break;
+        break;
       }
       reStr += "\\" + c;
       break;
 
+    case '\\':
```

Comment:
It doesn't seem like this is the place to unescape the "special characters" of zq search words.

Reply:
Okay, then where is the right place?